game.js

```diff
@@ -0,0 +1,143 @@
+sizeOfOneSquare, sizeOfOneSquare);
+        }
+    }
+}
+drawmap();
+var WidthOfWindow =  window.innerWidth*0.45, HeightOfWindow = 
+window.innerHeight*0.2;
+var flagForChangeStep = true, flagWinner = false;
+var arrayForO = [], arrayForX = [], countSteps = 0;
+
+function showCoords(evt) {
+    if (evt.clientX>=WidthOfWindow && evt.clientX 
+<=(WidthOfWindow+(sizeOfFoundation*sizeOfOneSquare)) && evt.clientY >= 
+HeightOfWindow && evt.clientY 
+<=(HeightOfWindow+(sizeOfFoundation*sizeOfOneSquare)) && flagWinner === 
+false) {
+        var coordinateOfX =evt.clientX, coordinateOfY =evt.clientY;
+        var numberi = Math.floor((coordinateOfX - 
+WidthOfWindow)/sizeOfOneSquare), numberj = Math.floor((coordinateOfY - 
+HeightOfWindow)/sizeOfOneSquare);
+        var ctx = canvas.getContext("2d");
+        var radiusOfCircle = sizeOfOneSquare/2;
+
+        if (flagForChangeStep) { 
+            var flagNotDrawX = true;
+            if (countSteps > 0) {
+                for (var i=0, len = arrayForO.length; i<len; i++) {
+                    if (numberi+numberj*sizeOfFoundation === 
+arrayForO[i] || numberi+numberj*sizeOfFoundation === arrayForX[i]) {
+                        flagNotDrawX = false;  
+                        break;
+                    }
+                }
+            }
+            if (flagNotDrawX === true) {
+                ctx.beginPath();              
+                ctx.lineWidth = "1";
+                ctx.strokeStyle = "black"; 
+                ctx.moveTo(0+numberi*sizeOfOneSquare, 
+0+numberj*sizeOfOneSquare);
+                ctx.lineTo(sizeOfOneSquare+numberi*sizeOfOneSquare, 
+sizeOfOneSquare+numberj*sizeOfOneSquare);
+                
+ctx.moveTo(sizeOfOneSquare+numberi*sizeOfOneSquare,0+numberj*sizeOfOneSquare);
+                
+ctx.lineTo(0+numberi*sizeOfOneSquare,sizeOfOneSquare+numberj*sizeOfOneSquare);
+                ctx.stroke();
+                flagForChangeStep = false;
+                arrayForX.push(numberi+numberj*sizeOfFoundation);
+                countSteps++;
+            }
+        }
+        else {
+            var flagNotDrawCircle = true;
+            for (var i=0, len = arrayForX.length; i<len; i++) {
+                if (numberi+numberj*sizeOfFoundation === arrayForX[i] || 
+numberi+numberj*sizeOfFoundation === arrayForO[i]) {
+                    flagNotDrawCircle = false;  
+                    break;
+                }
+            }
+            if (flagNotDrawCircle === true) {
+                ctx.beginPath();
+                
+ctx.arc(radiusOfCircle+numberi*sizeOfOneSquare,radiusOfCircle+numberj*sizeOfOneSquare,radiusOfCircle,0,2*Math.PI);
+                ctx.stroke();
+                flagForChangeStep = true;
+                arrayForO.push(numberi+numberj*sizeOfFoundation); 
+                countSteps++;  
+            }
+        }
+        winner();  
+    }  
+}
+
+function winner() {
+    var arrey = [
```

Comment:
I ALREADY WROTE IT. PLEASE, ANSWER ON MY COMMENTS OF FIX IT

Reply:
don't understand what you want